Agreements/Internal/TCCPT_25.txt

```diff
@@ -1,13 +1,3 @@
-COPX:EGBB:*:WOD:*:*:London TC CPT:London S25:21000:*:^WOD
-COPX:EGBE:*:WOD:*:*:London TC CPT:London S25:21000:*:^WOD
-COPX:EGSS:*:WOD:*:*:London TC CPT:London S25:21000:*:^WOD
-COPX:EGSC:*:WOD:*:*:London TC CPT:London S25:21000:*:^WOD
-
-COPX:EGBB:*:BENSU:*:*:London TC CPT:London S25:21000:*:^BENSU
-COPX:EGBE:*:BENSU:*:*:London TC CPT:London S25:21000:*:^BENSU
-COPX:EGSS:*:BENSU:*:*:London TC CPT:London S25:21000:*:^BENSU
-COPX:EGSC:*:BENSU:*:*:London TC CPT:London S25:21000:*:^BENSU
-
```

Comment:
You've made these changes here as well as in #4048 - all you need to do is add the EGBB and EGBE lines back and we can keep in just the one PR

Reply:
I'm confused. Add the lines back where?